doc/visual-programming/source/widgets/evaluate/liftcurve.md

```diff
@@ -7,31 +7,28 @@ Measures the performance of a chosen classifier against a random classifier.
+
+Cummulative gains chart shows the proportion of true positive instances (for example, the number of clients who accept the offer) as a function of the number of positive instances (the number of clients contacted), assuming the the instances are ordered according to the models probability of being positive (e.g. ranking of clients).
+
+![](images/LiftCurve-cummulative-gain.png)
+
+Lift curve shows the ratio between the proportion of true positive instances in the selection and the proportion of customers contacted. See [a tutorial for more details](https://medium.com/analytics-vidhya/understanding-lift-curve-b674d21e426).
 
 ![](images/LiftCurve-stamped.png)
 
-1. Choose the desired *Target class*. The default class is chosen alphabetically.
+1. Choose the desired *Target class*. The default is chosen alphabetically.
+2. Choose whether to observe lift curve or cummulative gains.
 2. If test results contain more than one classifier, the user can choose which curves she or he wants to see plotted. Click on a classifier to select or deselect the curve.
 3. *Show lift convex hull* plots a convex hull over lift curves for all classifiers (yellow curve). The curve shows the optimal classifier (or combination thereof) for each desired TP/P rate.
 4. Press *Save Image* if you want to save the created image to your computer in a .svg or .png format.
 5. Produce a report.
-6. 2-D pane with **P rate** (population) as x-axis and **TP rate** (true positives) as a y-axis. The diagonal line represents the behavior of a random classifier. Click and drag to move the pane and scroll in or out to zoom. Click on the "*A*" sign at the bottom left corner to realign the pane.
+6. A plot with **Lift** or **true positive rate** vs. **P rate**. The dashed line represents the behavior of a random classifier.
 
-**Note!** The perfect classifier would have a steep slope towards 1 until all
-classes are guessed correctly and then run straight along 1 on y-axis to
-(1,1).
 
 Example
 -------
 
-At the moment, the only widget which gives the right type of the signal needed by the **Lift Curve** is [Test & Score](../evaluate/testandscore.md).
-
-In the example below, we try to see the prediction quality for the class 'survived' on the *Titanic* dataset. We compared three different classifiers in the Test Learners widget and sent them to Lift Curve to see their performance against a random model. We see the [Tree](../model/tree.md) classifier is the best out of the three, since it best aligns with *lift convex hull*. We also see that its performance is the best for the first 30% of the population (in order of descending probability), which we can set as the threshold for optimal classification.
-
-![](images/LiftCurve-example.png)
-
-References
-----------
+The widgets that provide the right type of the signal needed by the **Lift Curve** (evaluation data) are [Test & Score](../evaluate/testandscore.md) and [Predictions](../evaluate/predictions.md).
```

Comment:
The Predictions part is true only for labelled data. Perhaps make this clear?

Reply:
Don't both?

Reply:
Yes, but Test & Score warns you about missing target variable, while Predictions doesn't. I don't know, I just think it would be clearer that way.

Reply:
I understand what you meant. But it's somehow clear that the data has to have a target variable, so I'd rather keep it short.